docs/member/strategist-profile.md

```diff
@@ -0,0 +1,64 @@
+# Strategist Profile
+
+## Introduction
+AlgoBulls provides a variety of trading strategies, but the brains behind these strategies are market experts who work extremely hard to create complex yet simple strategies. The Strategist profile will help you recognize the strategist's work and strategies. Let’s take a look at what the strategist profile offers
+
+## Strategist Details
+
+View the strategist's details such as Name, Bio & Contact Information.
+
+[![Strategist Profile](imgs/strategiest_profile_details.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/strategiest_profile_details.png)
+
+## Strategist Profile Summary
+
+Get the strategist’s profile summary details such as the total number of strategies, deployments, trades, wins, total loss, long trades, short trades, max gain, and more.
+
+[![Strategist Profile](imgs/strategiest_profile_summary.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/strategiest_profile_summary.png)
+
+
+
+## Explore the market experts’ strategies
```

Comment:
Are the number of lines left blank above this correct ?

Reply:
Extra lines removed